.github/workflows/test-application.yaml

```diff
@@ -0,0 +1,80 @@
+name: Test application
+
+on:
+    pull_request:
+    push:
+        branches:
+            - master
+            - develop
+            - '[0-9]+.[0-9]+'
+
+jobs:
+    test:
+        name: "Run tests with php ${{ matrix.php-version }}"
+        runs-on: ubuntu-18.04
+
+        env:
+            SYMFONY__DATABASE__PASSWORD: root
+
+        strategy:
+            fail-fast: false
+            matrix:
+                include:
+                    - php-version: '5.5'
+                      composer-flags: '--prefer-lowest --prefer-dist --no-interaction'
+                      tools: 'composer:v1'
+                      env:
+                          SYMFONY__DEPRECATIONS__HELPER: disabled
+                    - php-version: '7.4'
+                      composer-flags: '--prefer-dist --no-interaction'
+                      tools: 'composer:v2'
+                      env:
+                          SYMFONY__DEPRECATIONS__HELPER: weak
+
+        services:
+            mysql:
+                image: mysql:5.7
+                env:
+                    MYSQL_ROOT_PASSWORD: root
+                ports:
+                    - 3306:3306
+                options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=5
+
+        steps:
+            - name: Checkout project
+              uses: actions/checkout@v2
+
+            - name: Install and configure PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: ${{ matrix.php-version }}
+                  extensions: 'mysql, gd'
+                  tools: ${{ matrix.tools }}
+
+            - name: Get composer cache directory
+              id: composer-cache-dir
+              run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+            - name: Cache dependencies
+              uses: actions/cache@v2
+              id: composer-cache
+              with:
+                  path: ${{ steps.composer-cache-dir.outputs.dir }}
+                  key: ${{ runner.os }}-composer-${{ hashFiles('composer.lock') }}
+                  restore-keys: |
+                      ${{ runner.os }}-composer-
+
+            - name: Install dependencies
+              run: |
+                  composer validate --strict
+                  composer update ${{ matrix.composer-flags }}
+
+            - name: Create database
+              run: |
+                  Tests/app/console doctrine:database:create
+                  Tests/app/console doctrine:schema:update --force
+              env: ${{ matrix.env }}
```

Comment:
For now, this is okay, but for the develop branch this should be done by the `bootstrap-test-environment` script